cli/src/main.rs

```diff
@@ -88,6 +83,24 @@ async fn sign_up(pool: &PgPool, username_opt: Option<String>) -> color_eyre::Res
     Ok(())
 }
 
+async fn check_status(pool: &PgPool) -> color_eyre::Result<()> {
+    let user_path = Path::new("auth.txt");
+    if user_path.exists() {
+        let mut file = fs::File::open("auth.txt")?;
+        let mut contents = String::new();
+        file.read_to_string(&mut contents)?;
+        let user_id = Uuid::parse_str(contents.trim())?;
+
+        match db::get_username_by_id(pool, user_id).await? {
+            Some(username) => println!("Logged in with Username: {username}, User ID: {user_id}"),
+            None => println!("No user found with User ID: {user_id}"),
+        }
+    } else {
+        println!("Not logged in.");
+    }
+    Ok(())
+}
+
```

Comment:
This function checks for the auth.txt file in the root.
If there is 
it reads into the file and looks for the user_id in the DB to print out the user_id and user_name